R/process_lookup_postcode.R

```diff
@@ -4,18 +4,25 @@
 #' postcode lookup, it will return the final data
 #' but also write this out as a zsav and rds.
 #'
+#' @param spd_path Path to Scottish Postcode Directory.
+#' @param simd_path Path to SIMD lookup.
+#' @param locality_path Path to locality lookup.
+#'
 #' @param write_to_disk (optional) Should the data be written to disk default is
 #' `TRUE` i.e. write the data to disk.
```

Comment:
Just a general comment but we can use `@inheritParams` to inherit very common param descriptions like `write_to_disk` 
```suggestion
#' @inheritParams <top_level_fun>
```